src/components/App.tsx

```diff
@@ -2,21 +2,25 @@ import { getEncodedToken } from '@cashu/cashu-ts'
 import Button from '@comps/Button'
 import usePrompt from '@comps/hooks/Prompt'
 import { env } from '@consts'
+import { FiveMins } from '@consts/time'
 import { addAllMintIds, getBalance, getContacts, getMintsBalances, getMintsUrls, getPreferences, initDb, setPreferences } from '@db'
 import { fsInfo } from '@db/fs'
 import { l } from '@log'
 import MyModal from '@modal'
 import type { IInitialProps, IPreferences, ITokenInfo } from '@model'
```

Comment:
'IInitialProps' is defined but never used

Reply:
fixed